src/App.tsx

```diff
@@ -1,18 +1,27 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 import './App.scss';
+import { Tabs } from './Tabs';
 
 export const tabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
+
+  return (
+    <div className="App">
+      <h1>{`Selected tab is Tab ${selectedTab.title}`}</h1>
```

Comment:
```suggestion
      <h1>{`Selected tab is ${selectedTab.title}`}</h1>
```
you have "Tab" in "title"

Reply:
double tabs